### DISCUSSION TITLE
Provide an easy way to access previous versions docs

### DISCUSSION THREAD

**Discussion by adesurirey** [TanStack/query#2197] [Ideas] [answered]
React Query is moving very fast and I cannot afford to always upgrade to the latest version. 

Pains: 
- Currently, accessing a previous version documentation can only be done on github and is not trivial 
- When on the website, there is no way to know wich documentation version I'm looking at

Solution: 
- Add a versions dropdown on the website navbar

**Comment by TkDodo** [comment, marked as answer]
https://react-query-v2.tanstack.com/docs/overview

It’s right there with a link on the start page:
![140DB4F2-D333-4087-8B00-444750B9B543](https://user-images.githubusercontent.com/1021430/116120132-811c1e00-a6bf-11eb-8a2f-fdf203ddc8b1.jpeg)

yes, the search is not working. There are at least 3 closes issues about it :)
